cfme/tests/webui/test_general_ui.py

```diff
@@ -288,6 +289,47 @@ def test_add_button_on_provider_all_page(
     assert displayed_view.is_displayed
 
 
+@pytest.mark.tier(1)
+@pytest.mark.meta(automates=[1475553])
+def test_tls_openssl_verify_mode(temp_appliance_preconfig, request):
+    """
+    Polarion:
+        assignee: pvala
+        casecomponent: WebUI
+        caseimportance: medium
+        initialEstimate: 1/8h
+        testSteps:
+            1. Check if key `openssl_verify_mode` is present in the advanced configuration.
+            2. Navigate to Configuration and toggle `Start TLS Automatically`
+                of Outgoing SMTP E-mail Server.
+            3. Again check for the presence of `openssl_verify_mode` and check its value.
+        expectedResults:
+            1. Key must be absent.
+            2.
+            3. Key must be present and value must be None.
+
+    Bugzilla:
+        1475553
+    """
+    # take a fresh appliance where smtp has never been touched
+    appliance = temp_appliance_preconfig
+    # 1
+    assert "openssl_verify_mode" not in appliance.advanced_settings["smtp"]
+    # 2
+    appliance.server.settings.update_smtp_server({"start_tls": False})
+    # reset `Start TLS Automatically`
+    request.addfinalizer(
```

Comment:
Since this is on a temp_appliance, I don't think the finalizer is necessary here. The appliance will be destroyed after this test method is ran.